docs/AirGapImageConfiguration.md

```diff
@@ -49,7 +49,7 @@ spec:
 | Pipeline              | arg:shell-image                   | `IMAGE_PIPELINES_ARG__SHELL_IMAGE`                  |
 | Pipeline              | arg:shell-image-win               | `IMAGE_PIPELINES_ARG__SHELL_IMAGE_WIN`              |
 | Pipeline              | arg:workingdirinit-image          | `IMAGE_PIPELINES_ARG__WORKINGDIRINIT_IMAGE`         |
-| Pipeline              | controller                        | `IMAGE_PIPELINES_CONTROLLER`                        |
+| Pipeline              | controller (resolvers controller) | `IMAGE_PIPELINES_CONTROLLER`                        |
```

Comment:
Should we rename `IMAGE_PIPELINES_CONTROLLER` to `IMAGE_PIPELINES_TEKTON_RESOLVERS_CONTROLLER` too, WDYT?

Reply:
@pratap0007 we thought of doing that, but the environment key is generated dynamically, to change the environment key we have to change the container name in https://github.com/tektoncd/pipeline/blob/main/config/resolvers/resolvers-deployment.yaml#L65